Source/FDWaveformView.swift

```diff
@@ -158,16 +174,18 @@ open class FDWaveformView: UIView {
     }
 
     /// View for rendered waveform
-    fileprivate let imageView: UIImageView = {
+    lazy fileprivate var imageView: UIImageView = {
```

Comment:
Swift requires lazy properties to be var instead of let and the property was made lazy to allow for initializing the image view with the waves color.